specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,783 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppsJobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerAppsJobEnvelope",
+            "in": "body",
+            "description": "Properties used to create a container apps job",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ContainerAppsJob"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "No change performed on the Container Apps Job.",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppsJob"
+            }
+          },
+          "201": {
+            "description": "Container Apps Job create or update has been started.",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppsJob"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or Update Container Apps Job": {
+            "$ref": "./examples/ContainerAppsJob_CreateOrUpdate.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "ContainerAppsJobs"
+        ],
+        "summary": "Delete a Container Apps Job.",
+        "description": "Delete a Container Apps Job.",
+        "operationId": "ContainerAppsJob_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppsJobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Container Apps Job deleted successfully."
+          },
+          "202": {
+            "description": "Delete operation is in progress."
+          },
+          "204": {
```

Comment:
How is this situation different from 4xx user error? I think we should remove the 204 and bundle it up with 4xx with the appropriate message.

Reply:
@trajkobal added a 404 for this scenario.